docs/model_library/strategic_water_management/index.rst

```diff
@@ -664,13 +672,16 @@ Sets the flow capacity in a given pipeline during a given time period. Different
 
     \textcolor{red}{F_{l,l ̂,[t]}^{Capacity}} = \textcolor{green}{σ_{l,l ̂}^{Pipeline}}+∑_{d∈D}\textcolor{green}{δ_{d}^{Pipeline}}⋅(\textcolor{red}{y_{l,l ̂,d}^{Pipeline}}+\textcolor{red}{y_{l ̂,l,d}^{Pipeline}} )+\textcolor{red}{S_{l,l ̂}^{PipelineCapacity}}
 
-.. note:: Parameter :math:`δ_{d}^{Pipeline}` will be calculated as follows:
+.. note::
+
+    δ can be input by user or calculated. If the user chooses to calculate pipeline capacity, the parameter will be calculated by the equation below where :math:`{κ_{l,l}}` is Hazen-Williams constant and ω is Hazen-Williams exponent as per Cafaro & Grossmann (2020) and d represents the pipeline diameter as per the set d∈D.
```

Comment:
I'd recommend including the full citation to Cafaro and Grossman (2020) at the bottom of the page.

Reply:
implemented!